tools/hmcts-snl.postman_collection.json

```diff
@@ -1,29 +1,34 @@
 {
 	"info": {
-		"_postman_id": "d34e95bc-b58b-43e3-93f8-136df2aa4f9e",
 		"name": "hmcts-snl",
+		"_postman_id": "68a46e42-b525-4704-805b-151817b7154b",
 		"description": "Sample calls for the service",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
 	"item": [
 		{
-			"name": "current-state",
+			"name": "insert-availability",
 			"request": {
-				"method": "GET",
-				"header": [],
+				"method": "PUT",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					}
+				],
 				"body": {
 					"mode": "raw",
-					"raw": ""
+					"raw": "{\n\t\"type\": \"insert-availability\",\n\t\"data\": \"{\\\"id\\\":\\\"a1\\\",\\\"judgeId\\\":\\\"j1\\\",\\\"start\\\":\\\"2018-04-12T15:24:05.255+01:00\\\",\\\"duration\\\": 3600}\"\n}"
 				},
 				"url": {
-					"raw": "http://localhost:8091/exporthtml",
+					"raw": "http://localhost:8091/msg",
 					"protocol": "http",
 					"host": [
 						"localhost"
 					],
 					"port": "8091",
 					"path": [
-						"exporthtml"
```

Comment:
not sure why you removed 4 or 5 other calls like upsert session or exporthtml, please re-add